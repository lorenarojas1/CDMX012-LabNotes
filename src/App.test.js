import { render, screen } from '@testing-library/react';
import LoginPage from './pages/login';

const newLocal = screen.getByPlaceholderText('email@example.com');

test('renders learn react link', () => {
  render(<LoginPage />);
  const linkElement = newLocal;
  expect(linkElement).toBeInTheDocument();
});
