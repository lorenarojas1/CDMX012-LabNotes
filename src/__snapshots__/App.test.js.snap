// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test App renders learn react link 1`] = `
<div
  className="container-welcome"
>
  <section
    className="box-container"
  >
    <img
      alt="girl-icon"
      className="girl-image"
      src="girlIcon.svg"
    />
    <p
      className="descriptionWelcome"
    >
      The best app for
       
      <br />
       
      organize your
       
      <br />
       
      day...
    </p>
    <div
      className="box-buttons"
    >
      <h1
        className="titleWelcome"
      >
        JOIN OUR
         
        <br />
         
        TEAM
      </h1>
      <button
        className="btn-welcome"
        on={true}
        onClick={[Function]}
        type="button"
      >
        Login
      </button>
      <p
        className="orText"
      >
        -------------- or ---------------
      </p>
      <button
        className="btn-welcome"
        on={true}
        onClick={[Function]}
        type="button"
      >
        Sign up
      </button>
      <br />
      <button
        className="btn-google"
        on={true}
        onClick={[Function]}
        type="button"
      >
        <img
          alt="google-icon"
          className="icon-google"
          src="googleIcon.svg"
        />
        Continue with Google
      </button>
    </div>
  </section>
</div>
`;
